lib/pages/post_widgets/create_post.dart

```diff
@@ -15,26 +21,58 @@ class _CreatePostWidgetState extends State<CreatePostWidget> {
   TextEditingController postBodyInput = TextEditingController();
   TextEditingController postTitleInput = TextEditingController();
 
+  Post _post = Post(
+    communityName: '',
+    postId: '',
+    creator: '',
+    creationDate: 0,
+    title: '',
+    body: '',
+  );
+
   bool _isCreatingPost = false;
 
   doCreatePost() {
     setState(() {
       _isCreatingPost = true;
     });
 
-    final Future<Map<String, dynamic>> successfulMessage =
-        createPost(postTitleInput.text, postBodyInput.text, widget.communityName, widget.token);
+    final Future<Map<String, dynamic>> responseMessage = createPost(
+        postTitleInput.text,
+        postBodyInput.text,
+        widget.communityName,
+        widget.token);
 
-    successfulMessage.then((response) {
+    responseMessage.then((response) {
       if (response['status'] == true) {
+        Post post = Post(
+            communityName: widget.communityName,
+            postId: response['postId'],
+            creator: '',
```

Comment:
you might've forgotten this empty, can you add it?

Reply:
The creator isn't necessary here since the creator is actually set in the community widget. Since I'm using the post model some value is needed here. If I were to add the creator here as well then the user model/provider would also have to be added which I felt was unnecessary.